src/main/scala/it/bitrock/dvs/api/model/model.scala

```diff
@@ -1,13 +1,31 @@
 package it.bitrock.dvs.api.model
 
+import scala.concurrent.duration.FiniteDuration
+
 sealed abstract class WebSocketIncomeMessage(val `@type`: String)
-final case class CoordinatesBox(leftHighLat: Double, leftHighLon: Double, rightLowLat: Double, rightLowLon: Double)
-    extends WebSocketIncomeMessage("startFlightList")
+sealed trait FrequencyCommand {
```

Comment:
Do we need this?

Reply:
It is a common interface that indicate that this command should have a update rate